mailit/views.py

```diff
@@ -45,6 +46,19 @@ def get_success_url(self):
         return reverse('writeitinstance_template_update', subdomain=self.writeitinstance.slug)
 
 
+def send_error_email(email):
+    tb = traceback.format_exc()
```

Comment:
I think it's better to take the traceback where the exception is caught, and pass it into this function, otherwise the caller needs to know the precise context where this must be called.